src/dashboard/cd-dashboard-stats.vue

```diff
@@ -1,16 +1,49 @@
 <template>
   <div class="column">
     <div class="cd-dashboard-stats">
-      <h1 class="cd-dashboard-stats__header">Dojo Stats</h1>
+      <h1 class="cd-dashboard-stats__header">{{ $t('Dojo Stats') }}</h1>
+      <div class="cd-dashboard-stats__number">
+        <div class="cd-dashboard-stats__number-value"><span>{{ bookedChildren }}</span></div>
+        <div class="cd-dashboard-stats__description">{{ $t('kids attended your events') }}</div>
+      </div>
     </div>
   </div>
 </div>
 </template>
 
 <script>
+  import { mapGetters } from 'vuex';
+  import DojoService from '@/dojos/service';
+  import EventsService from '@/events/service';
+  
   export default {
     name: 'cd-dashboard-stats',
-    components: {
+    data() {
+      return {
+        dojos: null,
+        dojoId: null,
+        bookedChildren: 0,
+      };
+    },
+    computed: {
+      ...mapGetters(['loggedInUser']),
+    },
+    methods: {
+      async getDojos() {
+        this.dojos = (await DojoService.getUsersDojos(this.loggedInUser.id)).body;
+        // TODO : drop-down dojo selection
+        this.dojoId = (this.dojos[0]).dojoId;
```

Comment:
I don't want to manage rights at that level : i believe it should be the role of the whole dashboard

Reply:
Well, we need to something about it now. Options are basically
1. Handle it here
2. Pass usersdojos from dashboard, but have it filter to dojo-admins only for this component

Reply:
We don't, there is a ticket for handling visibility :) https://trello.com/c/lx7McY2n

Reply:
But that doesn't solve the problem in my initial scenario. It'd be made visible because they'd be a champion of Dojo B, but `this.dojos[0]` could be the usersdojo for Dojo A, which they do not have permissions to view.